src/main/java/org/isf/menu/gui/MainMenu.java

```diff
@@ -326,6 +345,35 @@ public void windowClosing(WindowEvent e) {
 		setVisible(true);
 	}
 
+	private HashMap<String, Boolean> retrieveActivatedModulesMap() {
```

Comment:
Shouldn't this be returning a `Map` type instead?

Reply:
yes, done, thank you